batch/launch_job.py

```diff
@@ -27,30 +27,39 @@
               help="The name of the AWS Batch Job Definition to use for the job")
 @click.option("-q", "--job-queue", "batch_job_queue", type=str, default="Batch-CovidPipeline", show_default=True,
               help="The name of the AWS Batch Job Queue to use for the job")
-def launch_batch(config_file, num_jobs, sims_per_job, dvc_target, s3_input_bucket, s3_output_bucket, batch_job_definition, batch_job_queue):
+@click.option("-p", "--parallelize-scenarios", "parallelize_scenarios", type=bool, default=False, show_default=True,
+              help="Launch a different batch job for each scenario")
+def launch_batch(config_file, num_jobs, sims_per_job, dvc_target, s3_input_bucket, s3_output_bucket, batch_job_definition, batch_job_queue, parallelize_scenarios):
 
-    raw_config = None
+    config = None
     with open(config_file) as f:
-        raw_config = f.read()
-    parsed_config = yaml.full_load(raw_config)
+        config = yaml.full_load(f)
```

Comment:
fyi (no need to change): the `confuse` library makes it easier to use yaml configs (handles empty/nulls, typechecking, etc), it's used in `SEIR.utils` as an example